config/amazingtrophies/trophies/magic/botania/04_elven_gateway.json

```diff
@@ -25,8 +25,8 @@
       ["       ", "       ", "       ", "  BDB  "],
       ["       ", "       ", "       ", " B   B "],
       ["       ", "       ", "       ", " D   D "],
-      ["C     C", "       ", "       ", " B   B "],
-      ["A     A", "       ", "       ", "  B~B  "]
+      ["       ", "       ", "       ", " B   B "],
+      ["       ", "       ", "       ", "  B~B  "]
```

Comment:
Could you remove the first three strings in the inner arrays and each outer blank space in the remaining strings?  Like this:
```json
      [" BDB "],
      ["B   B"],
      ["D   D"],
      ["B   B"],
      [" B~B "]
```
Otherwise the structure doesn't appear centered on the pedestal.